client/coral-embed-stream/src/Embed.js

```diff
@@ -144,7 +158,10 @@ class Embed extends Component {
       <div style={expandForLogin}>
         <div className="commentStream">
           <TabBar onChange={this.changeTab} activeTab={activeTab}>
-            <Tab><Count count={asset.totalCommentCount}/></Tab>
+            <Tab><Count
+              count={asset.totalCommentCount}
+              handleClick={this.handleClick}/>
```

Comment:
Yes. I see that. The issue is that each time it comes from other tab by click '# comments' it will refetch twice.